packages/config/config/devices/0x027a/zen75.json

```diff
@@ -0,0 +1,118 @@
+// 800 series (LR) starting with firmware 1.10
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN75",
+	"description": "Heavy Duty Switch",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xa005",
+			"zwaveAllianceId": 5179
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "On/Off",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#orientation_momentary"
+		},
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#led_indicator_four_options"
+		},
+		{
+			"#": "14",
+			"$import": "templates/zooz_template.json#led_indicator_color"
+		},
+		{
+			"#": "15",
+			"$import": "templates/zooz_template.json#led_indicator_brightness"
+		},
+		{
+			"#": "3",
+			"$import": "templates/zooz_template.json#auto_off_timer_0x_1x_3x_7x"
+		},
+		{
+			"#": "5",
+			"$import": "templates/zooz_template.json#auto_on_timer_0x_1x_3x_7x"
+		},
+		{
+			"#": "7[0x01]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Control Group 2 with Physical Switch",
+			"defaultValue": 1
+		},
+		{
+			"#": "7[0x02]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Control Group 2 with Connected 3-Way Switch",
+			"defaultValue": 1
+		},
+		{
+			"#": "7[0x04]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Control Group 2 with Z-Wave Commands",
+			"defaultValue": 1
+		},
+		{
+			"#": "7[0x08]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Control Group 2 with On/Off Timers",
+			"defaultValue": 1
+		},
+		{
+			"#": "8",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_on_prev"
+		},
+		{
+			"#": "9",
+			"$import": "templates/zooz_template.json#enable_scene_control"
+		},
+		{
+			"#": "11",
+			"$import": "~/templates/master_template.json#smart_switch_mode_0-2"
+		},
+		{
+			"#": "12",
+			"$import": "templates/zooz_template.json#3way_switch_type"
+		},
+		{
+			"#": "13",
+			"$import": "templates/zooz_template.json#smart_switch_mode_reporting"
+		},
+		{
+			"#": "16",
+			"$import": "templates/zooz_template.json#association_reports_binary"
+		},
+		{
+			"#": "17",
+			"$import": "templates/zooz_template.json#local_programming"
+		}
+	],
+	"compat": {
+		// The device sends Configuration CC info reports in 4-byte chunks, causing each query to block the network for roughly 1.5 seconds.
+		"skipConfigurationNameQuery": true,
+		"skipConfigurationInfoQuery": true
+	},
```

Comment:
Have you verified that this is actually necessary?

Reply:
I have not, I copied the closest match device I could find. Not really a zwave person.  More than happy to verify if I can figure it out.

Reply:
Thanks for checking. The compat flags were indeed not necessary. Looks like that bug is no longer present in newer Zooz firmwares.